db/migrate/20230405185153_create_user.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUser < ActiveRecord::Migration[7.0]
```

Comment:
- [ ] I ran across this error while trying to run the migration files. Kindly  check it out and fix it so that your app can be properly tested. 

You should delete multiple same migration files

![image](https://github.com/chiztechnology/blog-app/assets/63473584/a8d01dd5-df65-4ac0-a867-e000d765f904)

Reply:
thanks. I just erased the file.
I submit for another review then.

best regards